src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -878,6 +879,17 @@ public function providePathsForMakePathRelative()
             array('C:/aa/bb/../../cc', 'C:/aa/../dd/..', 'cc/'),
             array('C:/../aa/bb/cc', 'C:/aa/dd/..', 'bb/cc/'),
             array('C:/../../aa/../bb/cc', 'C:/aa/dd/..', '../bb/cc/'),
+            array('aa/bb', 'aa/cc', '../bb/'),
```

Comment:
Indeed, it should be removed (we did revert a similar test for the very same reason https://github.com/symfony/symfony/pull/22133#discussion_r107794666). It's safe to do it here I think.